libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/HmisProfiles.tsx

```diff
@@ -39,6 +39,9 @@ export default function HmisProfiles() {
     <CardWrapper
       {...(hmisProfiles.length !== 0 && { onReset })}
       title="HMIS IDs"
+      subtitle={
+        hmisProfiles.length !== 0 ? 'Fill in both HMIS ID Type and ID #' : ''
```

Comment:
`!==` compares are not very fun for me, even if necessary sometimes. would prefer:

```javascript
hmisProfiles.length ?  '': 'Fill in both HMIS ID Type and ID #' 
```
or 
```javascript
!!hmisProfiles.length ? 'Fill in both HMIS ID Type and ID #' : ''
```
but FWIW my preference is to define logic in function body, even if in this case it's pretty straight forward